### PR TITLE
feat: Improve MetaEvidence and hooks related

### PR DESCRIPTION
**Pull request by FedeMadoery**
# Description

Add a sample page to add it as evidenceDisplay on Kleros Court

Example: https://court.kleros.io/cases/269

### CODE REVIEW

File: pages/evidence/index.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    const message = JSON.parse(
+      window.location.search
+        .substring(1)
+        .replace(/%22/g, '"')
+        .replace(/%7B/g, '{')
+        .replace(/%3A/g, ':')
+        .replace(/%2C/g, ',')
+        .replace(/%7D/g, '}')
+        .replace(/%2F/g, '/'),
+    )
```
Comment (ramabit) [inline pages/evidence/index.tsx:49]: makes sense to refactor this in a function/util?

File: pages/evidence/index.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      disputeID,
+      arbitrableChainID,
+      arbitrableJsonRpcUrl,
+    })
+  }, [parameters])
+
+  return (
+    <>
+      <SafeSuspense>
+        <TBDataView
```
Comment (ramabit) [inline pages/evidence/index.tsx:73]: this data view applies only for evidences? if so, I propose a name change to "EvidenceDataView" or similar

File: src/hooks/subgraph/useBadgeByDisputeId.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+export default function useBadgeByDisputeId(appChainId?: number, disputeId?: string) {
+  const gql = appChainId
+    ? getSubgraphSdkByNetwork(appChainId as ChainsValues, SubgraphName.TheBadge)
+    : null
+
+  console.log('Parameters', {
+    appChainId,
+    disputeId,
+  })
```
Comment (ramabit) [inline src/hooks/subgraph/useBadgeByDisputeId.tsx:14]: please remove logs here

File: src/hooks/subgraph/useBadgeModelKlerosMetadata.tsx [hunk 4]
```diff
@@ -56,14 +56,30 @@ export function useRegistrationBadgeModelKlerosMetadata(
       : null,
     async ([,]) => {
       const badgeModelKlerosData = badgeModelKlerosMetadata.data
-
+      console.log(badgeModelKlerosData)
```
Comment (ramabit) [inline src/hooks/subgraph/useBadgeModelKlerosMetadata.tsx:59]: please remove this logs

File: src/hooks/subgraph/useBadgeModelKlerosMetadata.tsx [hunk 5]
```diff
@@ -56,14 +56,30 @@ export function useRegistrationBadgeModelKlerosMetadata(
...
       const res = await getFromIPFS<KlerosListStructure>(badgeModelKlerosData?.registrationUri)
+      console.log(res)
```
Comment (ramabit) [inline src/hooks/subgraph/useBadgeModelKlerosMetadata.tsx:61]: please remove this logs

File: src/hooks/subgraph/useBadgeModelKlerosMetadata.tsx [hunk 6]
```diff
@@ -105,9 +121,24 @@ export function useRemovalBadgeModelKlerosMetadata(
...
+          's3Url' in badgeModelKlerosRemovalMetadata.fileURI
+            ? badgeModelKlerosRemovalMetadata.fileURI.s3Url
+            : ''
+      }
+      if (
+        badgeModelKlerosRemovalMetadata?.fileHash &&
+        typeof badgeModelKlerosRemovalMetadata.fileHash === 'string'
+      ) {
+        badgeRemovalCriteria = badgeModelKlerosRemovalMetadata.fileHash
+      }
```
Comment (ramabit) [inline src/hooks/subgraph/useBadgeModelKlerosMetadata.tsx:136]: duplicated code, consider refactor here

File: src/hooks/subgraph/useBadgeModelKlerosMetadata.tsx [hunk 7]
```diff
@@ -56,14 +56,30 @@ export function useRegistrationBadgeModelKlerosMetadata(
...
+          's3Url' in badgeModelKlerosRegistrationMetadata.fileURI
+            ? badgeModelKlerosRegistrationMetadata.fileURI.s3Url
+            : ''
+      }
+      if (
+        badgeModelKlerosRegistrationMetadata?.fileHash &&
+        typeof badgeModelKlerosRegistrationMetadata.fileHash === 'string'
+      ) {
+        badgeRegistrationCriteria = badgeModelKlerosRegistrationMetadata.fileHash
+      }
```
Comment (ramabit) [inline src/hooks/subgraph/useBadgeModelKlerosMetadata.tsx:77]: duplicated code, consider refactor here

**Review by ramabit — CHANGES_REQUESTED** [review]

File: pages/evidence/index.tsx (hunk 2 above)
Comment (FedeMadoery) [reply pages/evidence/index.tsx:73]: Good call, rename to `CourtEvidenceDataView`

File: pages/evidence/index.tsx (hunk 1 above)
Comment (FedeMadoery) [reply pages/evidence/index.tsx:49]: Not sure, it wont be used in any other place soo

**Review by Agupane — CHANGES_REQUESTED** [review]
Can you add the translation files?

**Review by Agupane — APPROVED** [review]

**Review by ramabit — APPROVED** [review]

**Pull request merged**